atdll/atdll.cpp

```diff
@@ -38,6 +38,25 @@
 #define HOTKEY_CALIBRATION_MOD (MOD_SHIFT)
 #define HOTKEY_CALIBRATION_VK VK_F7
 
+// Hotkey for loading calibration
+#define HOTKEY_LOAD_ID 0xCAFD
+#define HOTKEY_LOAD_MOD (MOD_SHIFT)
+#define HOTKEY_LOAD_VK VK_F8
+
+// Hotkey for saving calibration
+#define HOTKEY_SAVE_ID 0xCAFC
+#define HOTKEY_SAVE_MOD (MOD_SHIFT)
+#define HOTKEY_SAVE_VK VK_F9
+
+// Default calibration file
+#define CALIBRATION_FILE "atx-calibration.conf"
```

Comment:
Just for consistency with existing files, can we make this something like `atcalibration.conf` to match the other files?

Reply:
Sounds good